tests/ExceptionBagTest.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace WtfPhp\JsonApiErrors\Tests;
+
+use Exception;
+use PHPUnit\Framework\TestCase;
+use WtfPhp\JsonApiErrors\ExceptionBag;
+use WtfPhp\JsonApiErrors\Tests\Fakes\TestResponse;
+
+class ExceptionBagTest extends TestCase
+{
+    protected ExceptionBag $bag;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+        $this->bag = new ExceptionBag();
+    }
+
+    /** @test */
+    public function itShouldAddOneThrowable()
```

Comment:
All the tests are laid out for Exceptions, but could we also add tests for a class that implements the Throwable-Interface?

Reply:
Please see the note box: https://www.php.net/manual/en/class.throwable.php

Reply:
Yeah so what about Tests for Errors?

Reply:
Done 😁